pkg/antctl/raw/multicluster/delete/member_cluster.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package delete
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	mcsscheme "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/scheme"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+type memberClusterOptions struct {
+	namespace  string
+	clusterSet string
+}
+
+var memberClusterOpts *memberClusterOptions
+
+var memberClusterExamples = strings.Trim(`
+Delete a member cluster in a ClusterSet
+$ antctl mc delete member-cluster <MEMBER_CLUSTER> -n <NAMESPACE> --cluster-set <CLUSTER_SET>
+`, "\n")
+
+func (o *memberClusterOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		o.namespace = metav1.NamespaceDefault
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("the ClusterSet cannot be empty")
+	}
+
+	return nil
+}
+
+func NewMemberClusterCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "member-cluster",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Delete a member-cluster in a ClusterSet",
+		Long:    "Delete a member-cluster in a ClusterSet",
+		Example: memberClusterExamples,
+		RunE:    memberClusterRunE,
+	}
+
+	o := &memberClusterOptions{}
+	memberClusterOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of member-cluster")
```

Comment:
no need to add `-` unless it's an option, please update all of them. thanks.

Reply:
done